packages/web3-eth-ens/src/ressources/ABI/Resolver.js

```diff
@@ -242,114 +495,153 @@ var RESOLVER = [
+            }
+        ],
+        "name": "multicall",
+        "outputs": [
+            {
+                "name": "results",
+                "type": "bytes[]"
+            }
+        ],
+        "payable": false,
+        "stateMutability": "nonpayable",
+        "type": "function"
+    },
+    {
+        "constant": true,
         "inputs": [
             {
-                "indexed": true,
                 "name": "node",
                 "type": "bytes32"
-            },
+            }
+        ],
+        "name": "content",
+        "outputs": [
             {
-                "indexed": false,
-                "name": "hash",
+                "name": "",
                 "type": "bytes32"
             }
         ],
-        "name": "ContentChanged",
-        "type": "event"
+        "payable": false,
+        "stateMutability": "view",
+        "type": "function"
     },
     {
-        "anonymous": false,
+        "constant": true,
         "inputs": [
             {
-                "indexed": true,
                 "name": "node",
                 "type": "bytes32"
-            },
+            }
+        ],
+        "name": "multihash",
+        "outputs": [
             {
-                "indexed": false,
-                "name": "name",
-                "type": "string"
+                "name": "",
+                "type": "bytes"
             }
         ],
-        "name": "NameChanged",
-        "type": "event"
+        "payable": false,
+        "stateMutability": "view",
+        "type": "function"
     },
     {
-        "anonymous": false,
+        "constant": false,
         "inputs": [
             {
-                "indexed": true,
                 "name": "node",
                 "type": "bytes32"
             },
             {
-                "indexed": true,
-                "name": "contentType",
-                "type": "uint256"
+                "name": "hash",
+                "type": "bytes32"
             }
         ],
-        "name": "ABIChanged",
-        "type": "event"
+        "name": "setContent",
+        "outputs": [],
+        "payable": false,
+        "stateMutability": "nonpayable",
+        "type": "function"
     },
     {
-        "anonymous": false,
+        "constant": false,
         "inputs": [
             {
-                "indexed": true,
                 "name": "node",
                 "type": "bytes32"
             },
             {
-                "indexed": false,
-                "name": "x",
-                "type": "bytes32"
-            },
-            {
-                "indexed": false,
-                "name": "y",
-                "type": "bytes32"
+                "name": "hash",
+                "type": "bytes"
             }
         ],
-        "name": "PubkeyChanged",
-        "type": "event"
+        "name": "setMultihash",
+        "outputs": [],
+        "payable": false,
+        "stateMutability": "nonpayable",
+        "type": "function"
     }
```

Comment:
Overall do I think are there some unrelated changes in this ABI.. we should verify every change or undo the non-required changes.

Reply:
Fixed in efd5cbf